src/Twig/FieldExtension.php

```diff
@@ -131,7 +133,7 @@ public function getSelected(Field\SelectField $field, $returnsingle = false, $re
     {
         $definition = $field->getDefinition();
 
-        if ($definition->get('type') !== 'select' || ! $field->isContentSelect() || ($field->isContentSelect() && $definition->get('mode') === 'format')) {
+        if ($definition->get('type') !== 'select' || ! $field->isContentSelect() || $definition->get('mode') === 'format') {
```

Comment:
@bobdenotter Doesn't this one have a different truth table compared to the one on the left? 🤔

Reply:
if `$field->isContentSelect()` is `false`, it falls through in the second part of the statement. It it _reaches_ the third part, we can already be sure that `$field->isContentSelect()` is `true`, so it is meaningless.